app/models/busstop_node.rb

```diff
@@ -194,10 +195,15 @@ def self.findRoute(startID,endID)
 		while true
 			#break and retrun `pathRoute` if we have reached the destinationNode.
 			if currentNode.id == destNode.id
+				currentNode.visited = false	#we might arrive on this through a different route.
```

Comment:
Line is too long. [83/80]<br>Use 2 (not 1) spaces for indentation.<br>Missing space after #.<br>Tab detected.